docs/blog/2016-10-06-structural-induction.lhs

```diff
@@ -108,7 +108,7 @@ Reflecting Lists into the Logic
 To talk about lists in the logic, we use the annotation
 
 \begin{code}
-{-@ LIQUID "--exact-data-cons" @-}
+{- LIQUID "--exact-data-cons" @-}
```

Comment:
Now that this is the default, perhaps it needs not be mentioned anymore in the post.